contributors/guide/contributor-cheatsheet/README.md

```diff
@@ -422,3 +422,4 @@ git push --force
 [approve]: https://prow.k8s.io/command-help#approve
 [GitHub Administration Team]: /github-management#github-administration-team
 [Kubernetes Patch Release]: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md
+[Testing locally]:https://jamesdefabia.github.io/docs/getting-started-guides/docker/
```

Comment:
The reference exists above as well. We shouldn't define it again and instead fix the existing reference to the correct link.

Reply:
What should I do next?
Could You suggest me some fixes since I'm new to open source

Reply:
I have updated the PR as per your suggestions @palnabarun. could you please review?